onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -705,15 +696,12 @@ DEFINE_INT4_UNPACK_TENSOR_IMPL(UInt4x2, TensorProto_DataType_UINT4)
 // Uses the model path to construct the full path for loading external data. In case when model_path is empty
 // it uses current directory.
 template <typename T>
-Status UnpackTensor(const ONNX_NAMESPACE::TensorProto& tensor, const Path& model_path,
+Status UnpackTensor(const ONNX_NAMESPACE::TensorProto& tensor, const std::filesystem::path& model_path,
                     /*out*/ T* p_data, size_t expected_num_elements) {
 #if !defined(ORT_MINIMAL_BUILD)
   if (HasExternalData(tensor)) {
-    return UnpackTensorWithExternalData(
-        tensor,
-        model_path.IsEmpty() ? nullptr : model_path.ParentPath().ToPathString().c_str(),
-        expected_num_elements,
-        p_data);
+    return UnpackTensorWithExternalData(tensor, model_path.empty() ? std::filesystem::path() : model_path.parent_path(),
```

Comment:
can this be simplified?

```suggestion
    return UnpackTensorWithExternalData(tensor, model_path.parent_path(),
```

Reply:
Thanks